src/highdicom/seg/sop.py

```diff
@@ -641,6 +641,12 @@ def __init__(
                 else:
                     planes = pixel_array
                 planes = planes.astype(np.uint8)
+                # It may happen that an boolean array is passed that should be
+                # interpreted as fractional segmentation type. In this case, we
+                # also need to stretch pixel valeus to 8-bit unsigned integer
+                # range by multiplying with the maximum fractional value.
```

Comment:
```suggestion
                # It may happen that a boolean array is passed that should be
                # interpreted as fractional segmentation type. In this case, we
                # also need to stretch pixel valeus to 8-bit unsigned integer
                # range by multiplying with the maximum fractional value.
```